content/articles/how-to-use-brotli-compression-method/index.md

```diff
@@ -0,0 +1,144 @@
+### Introduction
+Data compression is the process of encoding files to reduce their size, making them take up less space on your server and allowing them to be transferred to site clients more quickly. Compression reduces the amount of bandwith required for website rendering, which speeds up the process. tatic assets such as *.js*, *.css* and *.json* text files are commonly used in web application projects, with redundancy and compression playing an important role.
+
+### Prerequisites
+- Compression algorithms(Lossy & Lossless data compression)
+  
+### What is compression algoritm
+This is the process of reducing files in order to decrease their size and make them more portable.
+During the decompression process, it's also utilized to restore data to its original state.
+
+### What is Brotli
+Brotli is a lossless data compression algorithm that employs a variant of the *LZ77 algorithm*, as well as *Huffman coding* and *second-order context modelling*. Its compression ratio is comparable to the best general-purpose compression methods currently available.
+*LZ77 algorithm*,*Huffman coding* and *second-order context modelling* are lossless type of data compression algorithms.
+Lossy data compression algorithms include *JPEG* and *MP3*
+
+
+### How Brotli works
+First, download and build Brotli if it's not already installed on your system.
+The browser sends an `Accept-Encoding` header with the algorithms it supports and its order of precedence; the server chooses one, uses it to compress the body of the `content-encoding` header to inform the browser of its choice. 

```

Comment:
Again here, you have not addressed my suggestion.

I had asked you to add an image of the full header. it helps the reader understand better